src/service/atm_service.rb

```diff
@@ -0,0 +1,134 @@
+require_relative('../model/atm_unit.rb')
+require_relative('./transaction_service.rb')
+
+require 'date'
+
+module AtmService
+
+    def AtmService.deposit_money(amount, bank_account, atm_unit)
```

Comment:
do you need to specify `def AtmService.method_name`? Wouldn't `def method_name` do the same?